crates/biome_css_formatter/src/comments.rs

```diff
@@ -72,7 +72,7 @@ impl CommentStyle for CssCommentStyle {
         parse_suppression_comment(text)
             .filter_map(Result::ok)
             .flat_map(|suppression| suppression.categories)
-            .any(|(key, _)| key == category!("format"))
+            .any(|(key, _, _)| key == category!("format"))
```

Comment:
Nit: It may be better to use `..` here so it doesn't need to updated again the next time something is added. (Noticed this in some other places too.)

```suggestion
            .any(|(key, ..)| key == category!("format"))
```